django_project/minisass/settings.py

```diff
@@ -85,215 +68,108 @@
-    ('site_page.html', 'Site Page'),
-    ('map_page.html', 'Map Page'),
-    ('resource_page.html', 'Resources Page'),
-    ('footer_content.html', 'Footer Content Only'),
-)
-CMS_TEMPLATE_INHERITANCE = True
-
-CMS_PLACEHOLDER_CONF = {
-    'links': {
-            'plugins': ['LinkPlugin'],
-            'name': gettext("Links"),
-            'limits': {
-                'LinkPlugin': 10
-                }
-            },
-    'bottom-links': {
-            'plugins': ['LinkPlugin'],
-            'name': gettext("Bottom of Page Links"),
-            'limits': {
-                'LinkPlugin': 7,
-                }
-            }
-    }
-
-CMS_PLUGIN_CONTEXT_PROCESSORS = []
-CMS_PLUGIN_PROCESSORS = []
-CMS_APPHOOKS = (
-    'monitor.cms_app.MonitorApp',
-    # 'cmsplugin_blog.cms_app.BlogApphook'
-)
-PLACEHOLDER_FRONTEND_EDITING = True
-
-CMS_UNIHANDECODE_HOST = None
-CMS_UNIHANDECODE_VERSION = None
-# CMS_UNIHANDECODE_DECODERS = []
-# CMS_UNIHANDECODE_DEFAULT_DECODER = []
-
-CMS_MEDIA_PATH = 'cms/'
-CMS_MEDIA_ROOT = MEDIA_ROOT + CMS_MEDIA_PATH
-CMS_MEDIA_URL = MEDIA_URL + CMS_MEDIA_PATH
-CMS_PAGE_MEDIA_PATH = 'cms_page_media/'
-
-CMS_URL_OVERWRITE = True
-CMS_MENU_TITLE_OVERWRITE = False
-CMS_REDIRECTS = False
-CMS_SOFTROOT = False
-
-CMS_PERMISSIONS = True
-CMS_RAW_ID_USERS = True
-CMS_PUBLIC_FOR = 'all'
-
-CMS_SHOW_START_DATE = False
-CMS_SHOW_END_DATE = False
-CMS_SEO_FIELDS = False
-
-CMS_CACHE_DURATIONS = {
-        'content': 60,
-        'menus': 3600,
-        'permissions': 3600
-        }
-CMS_CACHE_PREFIX = 'minisass-prod'
-CMS_MAX_PAGE_PUBLISH_REVERSIONS = 25
-
-# WYM Editor settings
-# WYM_TOOLS
-# WYM_CONTAINERS
-# WYM_CLASSES
-# WYM_STYLES
-# WYM_STYLESHEET
-
-# filer, easy_thumbnails settings
-FILER_DEBUG = False
-FILER_ENABLE_LOGGING = True
-
-THUMBNAIL_PROCESSORS = (
-    'easy_thumbnails.processors.colorspace',
-    'easy_thumbnails.processors.autocrop',
-    'filer.thumbnail_processors.scale_and_crop_with_subject_location',
-    'easy_thumbnails.processors.filters',
-)
-
-# # blog plugin settings
-# JQUERY_JS = 'https://ajax.googleapis.com/ajax/libs/jquery/1.4.4/jquery.min.js'
-# JQUERY_UI_JS = 'https://ajax.googleapis.com/ajax/libs/jqueryui/1.8.12/jquery-ui.min.js'
-# JQUERY_UI_CSS = 'http://ajax.googleapis.com/ajax/libs/jqueryui/1.8.12/themes/smoothness/jquery-ui.css'
-
-# reCAPTCHA keys for minisass.org
-RECAPTCHA_PUBLIC_KEY = os.environ.get('RECAPTCHA_PUBLIC_KEY', '')
-RECAPTCHA_PRIVATE_KEY = os.environ.get('RECAPTCHA_PRIVATE_KEY', '')
-
-# tiny-MCE settings
-TINYMCE_DEFAULT_CONFIG = {
-   'theme': 'simple',
-   'relative_urls': False,
-   'width': 800,
-   'height': 600,
-   'resize': True
-}
-
 # email settings
-EMAIL_HOST = os.getenv('SMTP_HOST')
-EMAIL_PORT = os.getenv('SMTP_PORT')
-DEFAULT_FROM_EMAIL = os.getenv('SMTP_EMAIL')
-EMAIL_HOST_USER = os.getenv('SMTP_HOST_USER')
-EMAIL_HOST_PASSWORD = os.getenv('SMTP_HOST_PASSWORD')
-EMAIL_USE_TLS = os.getenv('SMTP_EMAIL_TLS')
+# EMAIL_HOST = os.getenv('SMTP_HOST')
```

Comment:
@tinashechiraya why do you commented these? you can use `os.environ.get('SMTP_HOST', '')` instead.